src/libos/src/net/syscalls.rs

```diff
@@ -135,18 +136,140 @@ pub async fn do_accept4(
     Ok(new_fd as isize)
 }
 
+pub async fn do_sendto(
+    fd: c_int,
+    base: *const c_void,
+    len: size_t,
+    flags: c_int,
+    addr: *const libc::sockaddr,
+    addr_len: libc::socklen_t,
+) -> Result<isize> {
+    if len == 0 {
+        return Ok(0);
+    }
```

Comment:
I agree.
It returns directly since it's the behavior of net.deprecated.

Since user can send 0 bytes and recv 0 bytes in UDP, we should not return here.